### PR TITLE
Create ReverseStack program in Java 

### PR DESCRIPTION
**Pull request by SamagraAg**
### Checklist
🚨 Please review this repository's [contribution guidelines](https://github.com/ShubhKotnala/Data_Structures_And_Algorithms/blob/main/CONTRIBUTIONS.md).

- [x] I've read and agree to the project's contribution guidelines.
- [x] I'm requesting to **pull a topic/feature/bugfix branch**.
- [x] I checked that my code additions will pass code linting checks and tests.
- [x] I have added a comment block to support my contribution, explaning about the problem along with test cases.

### Description
What does this change do and why?

Thank you!

### CODE REVIEW

**Comment by SamagraAg** [comment]
Please merge this ,
Its my first PR
@ShubhKotnala

File: java/stack/ReverseStack.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
+// Java code to reverse a
+// stack using recursion
+import java.util.Stack;
+ 
+class Test {
```
Comment (ShubhKotnala) [inline java/stack/ReverseStack.java:5]: Please update the class name to file name so it is easy to run in IDEs

**Review by ShubhKotnala — CHANGES_REQUESTED** [review]

**Comment by ShubhKotnala** [comment]
Hi @SamagraAg , I have added some comments. Did you get time to have a look at them?

**Comment by ShubhKotnala** [comment]
Closing as the Author seems inactive

**Pull request closed without merging**